### PR TITLE
planner, expression: Remove the default `asc` order in explain result

### LINKED ISSUE #16974: ucp: Remove the default `asc` order in explain result
## Description

```mysql
mysql> desc select * from t order by b;
+-------------------------+----------+-----------+---------------+--------------------------------+
| id                      | estRows  | task      | access object | operator info                  |
+-------------------------+----------+-----------+---------------+--------------------------------+
| Sort_4                  | 10000.00 | root      |               | test.t.b:asc                   |
| └─TableReader_8         | 10000.00 | root      |               | data:TableFullScan_7           |
|   └─TableFullScan_7     | 10000.00 | cop[tikv] | table:t       | keep order:false, stats:pseudo |
+-------------------------+----------+-----------+---------------+--------------------------------+
3 rows in set (0.00 sec)
```

The order information `asc` in `Sort_4` is redundant and can be removed because we all know the default order is ascending.

Related places:
1. order by in select, like `select * from t order by a`
2. order by in group_concat, like `select group_concat(order by a) from t`
3. order by in window function, like `select lag(a,1) over (order by a asc) from t`
## Score

* 50

## Mentor(s)

* @SunRunAway

Contact the mentors: **#tidb-challenge-program** channel in [TiDB Community](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE)  Slack Workspace

## Recommended Skills

* golang

## Learning Materials

* Chinese: [TiDB 精选技术讲解文章](https://github.com/pingcap/presentations/blob/master/hackathon-2019/reference-document-of-hackathon-2019.md)
* English: [Awesome-Database-Learning](https://github.com/pingcap/awesome-database-learning)

### PR DESCRIPTION
**Pull request by PengJi**
### What problem does this PR solve?

Close #16974

Problem Summary:

### What is changed and how it works?

What's Changed:
`asc` is the default value when we use `order by`, so, just print only `desc` in the explain，

How it Works:

### Related changes

- n/a

### Check List 

Tests 

- Unit test
- Integration test

Side effects

- n/a

### Release note 

- n/a

### CODE REVIEW

File: planner/core/explain.go [hunk 1]
```diff
@@ -542,7 +542,7 @@ func (p *PhysicalWindow) ExplainInfo() string {
 		}
 		buffer.WriteString("order by ")
 		for i, item := range p.OrderBy {
-			order := "asc"
+			var order string
```
Comment (SunRunAway) [inline planner/core/explain.go:545]: There will be an extra space that you should remove it.

Comment (PengJi) [reply planner/core/explain.go:545]: updated

**Comment by PengJi** [comment]
@SunRunAway @francis0407 PTAL

File: planner/core/testdata/plan_suite_unexported_out.json [hunk 2]
```diff
@@ -256,7 +256,7 @@
       "TableReader(Table(t))->Window(avg(cast(test.t.a, decimal(65,30) BINARY))->Column#14 over(partition by test.t.a))->Projection",
       "TableReader(Table(t))->Sort->Window(avg(cast(test.t.a, decimal(65,30) BINARY))->Column#14 over(partition by test.t.b))->Partition(execution info: concurrency:4, data source:TableReader_10)->Projection",
       "IndexReader(Index(t.f)[[NULL,+inf]])->Projection->Sort->Window(avg(cast(Column#16, decimal(65,4) BINARY))->Column#17 over(partition by Column#15))->Partition(execution info: concurrency:4, data source:Projection_8)->Projection",
-      "TableReader(Table(t))->Sort->Window(avg(cast(test.t.a, decimal(65,30) BINARY))->Column#14 over(order by test.t.a asc, test.t.b desc range between unbounded preceding and current row))->Projection",
+      "TableReader(Table(t))->Sort->Window(avg(cast(test.t.a, decimal(65,30) BINARY))->Column#14 over(order by test.t.a , test.t.b desc range between unbounded preceding and current row))->Projection",
```
Comment (SunRunAway) [inline planner/core/testdata/plan_suite_unexported_out.json:259]: > `order by test.t.a , test.t.b`

There's still an extra space after `test.t.a`.

Comment (PengJi) [reply planner/core/testdata/plan_suite_unexported_out.json]: already removed the space

**Comment by zz-jason** [comment]
@PengJi Thanks for your contribution! Please follow the PR title format to refactor the PR title:

```
PR Title Format:
1. pkg [, pkg2, pkg3]: what's changed
2. *: what's changed
```

**Pull request merged**